project1/styles.css

```diff
@@ -0,0 +1,85 @@
+body {
+    color: rgb(85, 85, 85);
+    background-color: rgb(230, 230, 230);
+    font-family: 'Cormorant Garamond', serif; 
+    
+ 
+     
+}
+
+.wrapper {
+    margin-right: 10%;
+    margin-left: 7%;
+    padding-right: 50px;
+    padding-left: 50px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    width: 80%; 
+    
+}
+
+h1 {
+    color: rgba(43, 65, 105, 0.87);
+    font-family: 'Lexend Mega', sans-serif;
+    font-size: 600%;
+    text-align: center; 
+    line-height: 2;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+}
+
+h2 {
+    color: rgb(102, 102, 102);
+    text-align: center;
+    font-size: 23px; 
+    line-height: 0.5;
+    font-weight: unset; 
+    margin-bottom: 300px;
+}
+blockquote { 
+    font-size: 35px;
+    letter-spacing: 0.5px;
+    line-height: 2;
+    color: rgb(102, 102, 102);
+    margin-top: 20px;
+    margin-bottom:50px;
+    margin-right: 25%;
+    margin-left: 25%;
+    font-weight: 600;
+    text-align: center;
+}
+
+p { 
+    font-size: 23px;
+    letter-spacing: 0.5px;
+    line-height: 2;
+    color: rgb(102, 102, 102);
+    margin-top: 20px;
+    margin-bottom: 20px;
+    margin-right: 10%;
+    margin-left: 10%;
+
+} 
+center {
```

Comment:
See my note in the HTML about the `center` element. If you change this to a `div`, remember to update this CSS to replace "center" with "div" and to add `text-align: center;` to this ruleset.

Reply:
You removed the `center` element from the HTML, so you can delete it here from the CSS, too (lines 61-65).